views/components/Sidebar/SidebarLink.jsx

```diff
@@ -0,0 +1,17 @@
+import { forwardRef } from 'react'
+import { NavLink } from 'react-router-dom'
+import './SidebarLink.css'
+
+const SidebarLink = forwardRef((props, ref) => (
+  <NavLink
+    ref={ref}
+    to={props.to}
+    className={({ isActive }) =>
+      isActive ? 'sidenav_link-active' : 'sidenav_link'
```

Comment:
using BEM to classname consistency, and the classnames package, I would suggest

```suggestion
      classnames('SidebarLink__link', { 'SidebarLink__link--active': isActive })
```

Reply:
[465c369](https://github.com/AMP-SCZ/dpdash/pull/458/commits/465c36952e0f8176957c65d89dfea1c6b86ca296)